trunk/SUAVE/Methods/Weights/Buildups/eHelicopter/prop.py

```diff
@@ -0,0 +1,225 @@
+# prop.py
```

Comment:
Propellers are already a thing in SUAVE. They are fully defined except weights. I suggest you input a propeller class and obtain all information from the provided class.

Reply:
The input can be within the method or the empty script. Chefs choice.

Reply:
Changed to take propeller class as function input.